resident-services/README.md

```diff
@@ -0,0 +1,72 @@
+
+* Resident Id Access Token Creation (Setup) : This thread contains 4 esignet api's Oauth details, Send OTP, Authentication, Authorization Code and 1 reisdent Login​ Redirect URI api.  After the login is successful, the resident will be redirected to the resident portal’s logged-in page. From Login​ Redirect URI api will get the id and access token which will be used further in the headers of resident service api's.
+
+* For execution purpose neeed to check for the mentioned properties: 
+   * esignet default properties: Update the value for the properties according to the execution setup. Perform the execution for esignet api's with redis setup. So check for the redis setup accordingly.
+          mosip.esignet.cache.size - Enabled while not using the redis setup. Can keep the cache size around more than 100k.
+          mosip.esignet.cache.expire-in-seconds (authcodegenerated) - 21600
+          mosip.esignet.access-token-expire-seconds - 86400
+          mosip.esignet.id-token-expire-seconds - 86400
+          spring.cache.type=redis - check for this property and enable the redis.
+   * application default properties: Update the value for the below property.
+          mosip.kernel.otp.expiry-time - 86400
+   * id-authentication default properties: Update the value for the below properties.
+          otp.request.flooding.max-count - 100000
+
+          
+
+### Data prerequisite
+
+* List of VID's as per environment which is valid and will be prepared from the above mentioned create identities setup.
+
+### Execution points for Resident Service endpoints
+
+* The id and access token generated from the Login​ Redirect URI api will be stored in a file and will be used in the headers of below mentioned api's :
+
+    * Authorize​ Admin​ Validate Token 
+    * VID Policy
+    * Generate VID
+    * Get VIDs of the Resident
+    * Service History 
+    * Auth Lock Unlock 
+    * Auth Lock Status 
+    * Request Card VID 
+    * Service History Download 
+    * Revoke VID
+
+* Login​ Redirect URI (Preparation) - This thread contains set of 4 esignet api's Oauth details, Send OTP, Authentication, Authorization Code. The code generated will be stored in a file and can't be used for multiple runs as code can only be used once. 
+* Login​ Redirect URI (Execution) - In this api will use the code generated from the above esignet api. The total preparation samples must be equal or higher in number.
+
+* Revoke VID (Preparation) - This thread contains Generate VID api from which we will get a VID which needs to be revoked.
+* Revoke VID (Execution) - The generated VID can only be used once to be revoked so the samples created in the preparation must be equal or higher in number.
```

Comment:
The question of can we reuse the prepartion again is not answered.
This to be taken care for all APIs

Reply:
For both Login​ Redirect URI and Revoke VID already mentioned that the preparation can only be used once.
Mentioned for the other APIs too.